.travis.yml

```diff
@@ -2,6 +2,10 @@ sudo: false
 language: python
 python:
     - "3.6"
+services:
+    - docker
+before_install:
+    - docker pull jacobcallahan/genvirt
```

Comment:
I would suggest to use light image here
```
# docker images
REPOSITORY                        TAG                 IMAGE ID            CREATED             SIZE
docker.io/alpine                  latest              965ea09ff2eb        3 weeks ago         5.55 MB
docker.io/jacobcallahan/genvirt   latest              0cff1e445b4b        13 months ago       303 MB
```

Reply:
I am also curious to know why we need it here when you have added test for it. Can't test pull it?

Reply:
So the reason we went with this image is because future tests should exercise additional functionality that alpine doesn't have (registration, etc..). Additionally, it is an image that is under our control, so feature set is stable and expandable when needed.

Reply:
As far as explicitly pulling it here, I believe Travis was having issues with it only pulling in the test.

Reply:
This PR don't have any additional functionality tests such as registration so I would prefer to have any related changes in future PR itself. Also doing registration won't be so straight forward as it includes handling of credentials as well.

Reply:
Also I confirmed that travis yml needs only. This will allow tests to pull image. Logs - https://travis-ci.com/ntkathole/test/builds/136263059
```
services:
    - docker
```
We can remove 
```
before_install:
    - docker pull jacobcallahan/genvirt
```

Reply:
@ntkathole I've removed the ```before_install:``` portion before and travis still spat out errors about pulling the image.

Reply:
> This PR don't have any additional functionality tests such as registration so I would prefer to have any related changes in future PR itself. Also doing registration won't be so straight forward as it includes handling of credentials as well.

Hello, I have pull request open that does registration https://github.com/SatelliteQE/robottelo/pull/7469

Reply:
@swadeley Its not about adding product test, but its about robottelo unit tests.
@latran I am ok to keep before_install: section, but, I still think we should go with lighter image as of now. We acan think of using genvirt image when unit tests for additional functionalities such as registration will be added.

Reply:
@latran go ahead and try the switch to alpine (above) and see if the robottelo tests work as expected.

Reply:
Switched to alpine.  Please see changes.